data/pankajktech.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "Pankaj Kumar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web Developer, Student",
+  "tags": ["Open Source", "Content Writer", "Blogging", "Web3"],
+  "avatar": "https://github.com/pankajktech.png",
+  
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/PankajThakurOff",
+      "icon": "FaTwitter"
+    },
+
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/pankajthakuroff/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/pankajktech/",
+      "icon": "FaLinkedin"
+    }
```

There is another missing comma here. 
```suggestion
    },
```

---

Done